panic: missing context

<!-- Provide a general summary of the issue in the Title above -->
TestStorageGroup system tests are falling:

```
RuntimeError: Command: neofs-cli --config /home/runner/work/neofs-node/neofs-node/neofs-testcases/wallet_config.yml storagegroup put --rpc-endpoint 's01.neofs.devenv:8080' --wallet '/home/runner/work/neofs-node/neofs-node/neofs-testcases/TemporaryDir/8c2db297-27c4-4352-824a-2e6cd51f246b.json' --cid '93oPdTRLyiHiEvgQx72aBwJ4sh3sC1bDq1aFFjdjBWvT' --members 'VHV8rmtL8n2h4zPXaKAyoyhck9RFToxryDKT4aaDKZZ' --lifetime 10
Error:
return code: 2
output: panic: missing context

goroutine 1 [running]:
github.com/nspcc-dev/neofs-sdk-go/client.(*Client).ObjectHead(0xc00130a120, {0x0?, 0x0}, {{{0x0, 0x0, 0x0, {0x0, 0x0, 0x0}, 0x0, ...}, ...}, ...})
	github.com/nspcc-dev/neofs-sdk-go@v1.0.0-rc.8/client/object_get.go:417 +0x4eb
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client.HeadObject({0x0, 0x0}, {{{0xc00130a120}, {0x0}, 0x0, 0x0, {0x164f030, 0x0, 0x0}}, {{{0x77, ...}, ...}}, ...})
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/client/client.go:620 +0x3ee
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/storagegroup.sgHeadReceiver.Head({{0x0, 0x0}, 0x160db40, 0xc0002b8270, 0xc0000101b0, {{{0xc00130a120}, {0x0}, 0x0, 0x0, {0x164f030, ...}}, ...}}, ...)
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/storagegroup/put.go:140 +0x99
github.com/nspcc-dev/neofs-node/pkg/services/object/util.traverseSplitChain({0xf9a200, 0xc00133eb00}, {{0x77, 0x95, 0x1c, 0x4c, 0x12, 0xe, 0x45, 0xc0, ...}, ...}, ...)
	github.com/nspcc-dev/neofs-node/pkg/services/object/util/chain.go:76 +0x7d
github.com/nspcc-dev/neofs-node/pkg/services/object/util.TraverseSplitChain(...)
	github.com/nspcc-dev/neofs-node/pkg/services/object/util/chain.go:71
github.com/nspcc-dev/neofs-node/pkg/services/object/util.IterateSplitLeaves({0xf9a200, 0xc00133eb00}, {{0x77, 0x95, 0x1c, 0x4c, 0x12, 0xe, 0x45, 0xc0, ...}, ...}, ...)
	github.com/nspcc-dev/neofs-node/pkg/services/object/util/chain.go:43 +0x165
github.com/nspcc-dev/neofs-node/pkg/services/object/util.IterateAllSplitLeaves({0xf9a200, 0xc00133eb00}, {{0x77, 0x95, 0x1c, 0x4c, 0x12, 0xe, 0x45, 0xc0, ...}, ...}, ...)
	github.com/nspcc-dev/neofs-node/pkg/services/object/util/chain.go:28 +0xb3
github.com/nspcc-dev/neofs-node/pkg/services/object_manager/storagegroup.CollectMembers({0xf9a200, 0xc00133eb00}, {0x77, 0x95, 0x1c, 0x4c, 0x12, 0xe, 0x45, 0xc0, ...}, ...)
	github.com/nspcc-dev/neofs-node/pkg/services/object_manager/storagegroup/collect.go:31 +0x23b
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/storagegroup.putSG(0x160db40, {0xda0341?, 0xc?, 0xc?})
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/storagegroup/put.go:97 +0x7fe
github.com/spf13/cobra.(*Command).execute(0x160db40, {0xc000000900, 0xc, 0xc})
	github.com/spf13/cobra@v1.6.1/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x16048a0)
	github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.6.1/command.go:968
github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules.Execute()
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/modules/root.go:52 +0x25
main.main()
	github.com/nspcc-dev/neofs-node/cmd/neofs-cli/main.go:6 +0x17
```

See:
https://nspcc-dev.github.io/neofs-node/232/#categories/91a4766abd98e1e5227bc2ba4928463e/cf1ed29dfb80ffdb/

## Expected Behavior
No panics

## Current Behavior
panic: missing context

## Possible Solution
None

## Steps to Reproduce (for bugs)
Run TestStorageGroup system tests

## Context
System tests are falling

## Regression
Yes

## Your Environment
Test run: https://github.com/nspcc-dev/neofs-node/actions/runs/5323708445
```
neo-go 0.101.1
neofs-s3-authmate 0.27.1
neofs-cli 8abbe353
neofs-adm 8abbe353
neofs-s3-gw 0.27.1
neofs-ir 8abbe353 
neofs-lens 8abbe353 
AWS aws-cli/2.11.27 
Python/3.11.3
Linux/5.15.0-1039-azure exe/x86_64.ubuntu.22 prompt/off
```

--- a/cmd/neofs-cli/modules/storagegroup/put.go
+++ b/cmd/neofs-cli/modules/storagegroup/put.go
@@ -95,6 +95,7 @@ func putSG(cmd *cobra.Command, _ []string) {
 	headPrm.SetClient(cli)
 
 	sg, err := storagegroup.CollectMembers(sgHeadReceiver{
+		ctx:     ctx,
 		cmd:     cmd,
 		key:     pk,
 		ownerID: &ownerID,

